### ISSUE TITLE
Scroll bars for the window panes disappear after find command

### ISSUE THREAD

**Issue by wakululuu** [wakululuu/ped#2]
Executed `find 0` but as can be seen from the screenshot, the scroll bars for the Admin and Ip have disappeared, preventing the user from seeing the cut-off words.
![Screenshot 2020-10-30 at 12.22.55 PM.png](https://raw.githubusercontent.com/wakululuu/ped/main/files/4ef21b08-b1b0-4ddd-acb9-cd687f9a68e2.png)

**Comment by onglijin** [comment]
Hello! The scroll bars disappears because there are fewer tasks than the size of the window for admin and ip (aka topic and tp > 2 tasks). Thanks for noticing this detail though!

**Comment by wakululuu** [comment]
Makes sense that the vertical scroll bar disappears since there are fewer than 2 tasks. But the horizontal one should still be there in this case since the description can still exceed the width of the window.
